### ISSUE TITLE
Include Link to Subgraph on Everest

### ISSUE THREAD

**Issue by XingerZ** [graphprotocol/mission-control-curator#30]
For projects with available subgraph, it may be a good idea to allow community or team to add subgraph.

**Comment by alterseny93** [comment]
I support this proposal. it would be great to see which projects already have their own subgraphs.

**Comment by yanivtal** [comment]
Yes for sure. We need to work on composability and making it easier to add fields. Linking to one or many subgraphs makes a lot of sense 👍
